aries_cloudagent/indy/sdk/tests/test_verifier.py

```diff
@@ -322,20 +322,31 @@ async def test_verify_presentation(self, mock_verify):
         ) as mock_pre_verify, async_mock.patch.object(
             self.verifier, "non_revoc_intervals", async_mock.MagicMock()
         ) as mock_non_revox:
+            INDY_PROOF_REQ_X = deepcopy(INDY_PROOF_REQ_PRED_NAMES)
             verified = await self.verifier.verify_presentation(
-                INDY_PROOF_REQ_PRED_NAMES,
+                deepcopy(INDY_PROOF_REQ_X),
```

Comment:
Might not need this second `deepcopy` statement.

Reply:
Fixed (oops)